requirements/requirements_test.txt

```diff
@@ -1,4 +1,4 @@
-black==19.10b0
+black~=24.3
```

Comment:
Shouldn't this be 24.3?

Reply:
`black~=24.3` means (`black>=24.3` and `black<25`). Major version is stable, so `black~=24.3` is safe.